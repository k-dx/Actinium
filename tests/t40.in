let a = [1,[20, 21],3, fn() { return 42; }];
print a[3]();
let b = a[3]();
a[3] = 100;
print a;