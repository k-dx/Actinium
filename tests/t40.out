42
[1, [20, 21], 3, 100]